libpod/container_internal.go

```diff
@@ -898,14 +952,32 @@ func (c *Container) generateHosts() (string, error) {
 	return c.writeStringToRundir("hosts", hosts)
 }
 
+func (c *Container) getRunDir() string {
```

Comment:
If we're using a separate runDir from the one c/storage gives us, we should just store it in c.state.RunDir instead of calling this repeatedly.

Reply:
we need both.  The idea is that the new directory will be used only for a bind mount from the original directory that containers/storage gave us so we don't really store any file in the new path.